suricata.yaml.in

```diff
@@ -731,6 +731,9 @@ af-packet:
     #copy-iface: eth1
     #  For eBPF and XDP setup including bypass, filter and load balancing, please
     #  see doc/userguide/capture-hardware/ebpf-xdp.rst for more info.
+    # Disable HW timestamp support. HW timestamps can be unreliable, see
+    # https://redmine.openinfosecfoundation.org/issues/7585
+    #disable-hwtimestamp: true
```

Comment:
Just noticing now, as I look at it in backport context. This would suggest that it's disabled by default? Typically the example is set to the default. And we should probably also state what the default is.